### PR TITLE
Add ConsumeBili source

### PR DESCRIPTION
**Pull request by hoangvu12**
Bilibili with 1080p and 4K quality :)

### CODE REVIEW

File: app/src/main/java/ani/saikou/parsers/anime/Bilibili.kt [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    }
+
+    override suspend fun getVideoExtractor(server: VideoServer): VideoExtractor {
+        return BilibiliExtractor(server)
+    }
+
+    class BilibiliExtractor(override val server: VideoServer) : VideoExtractor() {
+        override suspend fun extract(): VideoContainer {
+            val extra = server.extraData as Map<*, *>
+
```
Comment (scrazzz) [inline app/src/main/java/ani/saikou/parsers/anime/Bilibili.kt:66]: Why's there a lot of unwanted newlines? It's not just here, check other methods too.

File: app/src/main/java/ani/saikou/parsers/anime/Bilibili.kt [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                link = it.id.toString(),
+                coverUrl = FileUrl(it.image)
+            )
+        }
+    }
+
+
+    @Serializable
+    data class SearchResponse(
+        @SerialName("totalResults") var totalResults: Int?,
```
Comment (scrazzz) [inline app/src/main/java/ani/saikou/parsers/anime/Bilibili.kt:95]: Please change all of these to `val`s. `var` isn't used here normally.

**Review by scrazzz — CHANGES_REQUESTED** [review]

File: app/src/main/java/ani/saikou/parsers/anime/Bilibili.kt (hunk 1 above)
Comment (hoangvu12) [reply app/src/main/java/ani/saikou/parsers/anime/Bilibili.kt:66]: > Why's there a lot of unwanted newlines? It's not just here, check other methods too.

Uh for better reading I think? I usually do this on other programming language so this is a habit.

Comment (scrazzz) [reply app/src/main/java/ani/saikou/parsers/anime/Bilibili.kt:66]: It's readable without the newlines. Check other sources. It doesn't have inconsistent newlines on every functions.

**Comment by brahmkshatriya** [comment]
We consume the Cat

**Pull request merged**